komodo-relational-commands/src/test/java/org/komodo/relational/commands/server/ServerSetPropertyCommandTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2014 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.komodo.relational.commands.server;
+
+import static org.junit.Assert.assertTrue;
+import java.util.ArrayList;
+import org.junit.Test;
+import org.komodo.shell.api.CommandResult;
+
+/**
+ * Test Class to test {@link ServerSetPropertyCommand}.
+ */
+@SuppressWarnings( {"javadoc", "nls"} )
+public final class ServerSetPropertyCommandTest extends AbstractServerCommandTest {
+
+    @Test
+    public void testSetProperty1() throws Exception {
+        final String[] commands = {
+            "server-set-property adminPort 8765",
+            "server-show-properties" };
+        final CommandResult result = execute( commands );
+        assertCommandResultOk(result);
+
+        String writerOutput = getCommandOutput();
```

Comment:
I think I would check to make sure the global property had the right value here instead of checking the command output.